app/src/main/java/org/jellyfin/androidtv/data/eventhandling/TvApiEventListener.java

```diff
@@ -133,24 +133,21 @@ else if(playbackController != null && mediaManager.hasVideoQueueItems() && playb
     @Override
     public void onBrowseCommand(ApiClient client, BrowseRequest command) {
         Timber.d("Browse command received");
+        if (command.getItemId() == null || command.getItemId().equals("")) return;
```

Comment:
In Java code use one of these:

https://github.com/jellyfin/jellyfin-androidtv/blob/0de3437e4e8d06e849f667ba683506a6423acac5/app/src/main/java/org/jellyfin/androidtv/util/Utils.java#L80-L86